modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -61,17 +62,22 @@ object types {
     remainingWeight: Weight
   )
 
+  case class OneTimeReward(epoch: EpochProgress, address: Address, txnAmount: TransactionAmount)
```

Comment:
`txnAmount` -> just `amount`? minor

Reply:
@marcinwadon If I do that then I have to worry about runtime validation when I convert it. Is there a reason we shouldn't use TxnAmount?

Reply:
I think he meant the name of the variable, not the class name?